.github/workflows/js-checks.yml

```
@@ -0,0 +1,15 @@
+name: JS Checks
+
+on:
+  pull_request:
+  push:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  run-checks-and-tests:
+    uses: bloq/actions/.github/workflows/js-checks.yml@v1
```

shouldn't we use hemilabs here?

```suggestion
    uses: hemilabs/actions/.github/workflows/js-checks.yml@v1
```

---

Not until https://github.com/hemilabs/actions/pull/22 is merged.

---

:agitate:

---

let's at least create an issue so we don't "merge and forget" to later update these actions (unless it's your secret plan to continue using them 😆 )

---

It is! 🤫

---

I still think we should go with the Hemilabs action

```suggestion
    uses: hemilabs/actions/.github/workflows/js-checks.yml@v1
```
From the perspective/usage of this action, the hemilabs action works exactly the same as the Bloq one, with no difference.   
If any internal problem (like the discrepancies we're having between hashes vs tags) exists, it should be solved in those actions. For all purposes, I think the benefits of automatically pulling fixes and improvements from hemilabs/actions are larger than the discussion we're having there. Besides, once solved (regardless of the outcome), this package will pick it up automatically, instead of us having to remember to update it.

Moreover, the `bloq/actions` also has a different team's setup, so reviews may be misleading as they're not reviewed by the same people that this org.

---

@gndelia `hemilabs/actions/.github/workflows/js-checks.yml` has a reference to `hemilabs/actions/setup-node-env@main`, which is not acceptable 😕